app/templates/reminders/reminder/edit.hbs

```diff
@@ -0,0 +1 @@
+{{ edit-reminder model=model title=model.title }}
```

Comment:
We were working with Taylor on this and thought it would solve a problem we were having, but sadly it didn't. Will make title consistent with the rest 👍